sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -311,6 +311,7 @@ private static RecognizeRequestInternal CreateRecognizeRequest(CallMediaRecogniz
                 RecognizeOptionsInternal recognizeConfigurationsInternal = new RecognizeOptionsInternal(CommunicationIdentifierSerializer.Serialize(recognizeChoiceOptions.TargetParticipant))
                 {
                     InterruptPrompt = recognizeChoiceOptions.InterruptPrompt,
+                    InitialSilenceTimeoutInSeconds = (int)recognizeChoiceOptions.InitialSilenceTimeout.TotalSeconds
```

Comment:
can't remember it clearly but, are we 100% sure that InitialSilenceTimout cannot be null?

Reply:
@cochi2 , Yes InitialSilenceTimout can be null, The issue is even if we are setting the value on SDK the InitialSilenceTimout property is not getting passed to PMA
For example i have set the timeout to 40secs and on PMA i dont see the property getting passed 
PMA Logs: https://portal.microsoftgeneva.com/s/5C10F776

Reply:
If `InitialSilenceTimeout` can be null, this line will throw a NullPointerException.